xml/System.IO.Pipelines/FlushResult.xml

```diff
@@ -39,9 +39,9 @@
         <Parameter Name="isCompleted" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="isCanceled">To be added.</param>
-        <param name="isCompleted">To be added.</param>
-        <summary>Creates a new instance of <see cref="T:System.IO.Pipelines.FlushResult" /> setting <see cref="P:System.IO.Pipelines.FlushResult.IsCanceled" /> and <see cref="P:System.IO.Pipelines.FlushResult.IsCompleted" /> flags.</summary>
+        <param name="isCanceled"><see langword="true" /> to indicate the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation that produced this <see cref="T:System.IO.Pipelines.FlushResult" /> was canceled by <see cref="M:System.IO.Pipelines.PipeWriter.CancelPendingFlush()" />; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
        <param name="isCanceled"><see langword="true" /> to indicate the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation that produced this <see cref="T:System.IO.Pipelines.FlushResult" /> was canceled by <see cref="M:System.IO.Pipelines.PipeWriter.CancelPendingFlush" />; otherwise, <see langword="false" />.</param>
```